src/Admin/OptionsMenu.php

```diff
@@ -185,6 +185,24 @@ public function screen_handle() {
 		return sprintf( 'toplevel_page_%s', $this->get_menu_slug() );
 	}
 
+	/**
+	 * Gets analytics vendors list from data directory.
+	 */
+	public function get_analytics_vendors() {
+		$vendors_list_file = AMP__DIR__ . '/data/analytics-vendors-list.json';
+
+		if ( ! file_exists( $vendors_list_file ) ) {
+			return []; // @codeCoverageIgnore
+		}
+
+		$vendors_list = file_get_contents( $vendors_list_file );
+		$vendors_list = json_decode( $vendors_list, true );
```

Comment:
Instead of storing the file as JSON, it could instead be stored as a PHP array in the same way that `update-extension-files.js` is doing. This would be faster to load.

Reply:
Agree, I did follow the Issue [description](https://github.com/ampproject/amp-wp/issues/6367#issuecomment-859030499)
for the reference and implemented the JSON file.

Will surely make the changes as suggested now.

Reply:
Case in point: https://twitter.com/schlessera/status/1492250702000832514?t=S4lnlmBMoFPB-ou24kjEug&s=19

Reply:
Resolved
Converted it to a PHP Array